readme.md

```diff
@@ -665,3 +667,46 @@ cancels an order.
     status: true|false
 }
 ```
+
+
+## GET /v0/tradeHistory
+`public endpoint`
+
+Gets trade history for a market.
+#### parameters
+* market - Symbol of a market
+* page - page of results (default 1)
+* per_page - number of results per page (default 10, max 100)
+* since - ISO 8601 start date (default now, e.g., 2018-03-07T16:31:27Z)
```

Comment:
default for since is not now. That would return no results. I reckon we don't mention a default for since but indicate the format.
Obviously if since is not specified all results are returned

Reply:
Oh right, good call. Will fix.